kotlin-native-sharing.md

```diff
@@ -27,7 +27,8 @@ For interoperability with code that is using Kotlin/Native
 [Worker](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.native.concurrent/-worker/index.html)
 API you can get a reference to single-threaded dispacher's worker using its [SingleThreadDispatcher.worker] property.
 
-A [Default][Dispatchers.Default] dispatcher on Kotlin/Native contains a single background thread. 
+A [Default][Dispatchers.Default] dispatcher on Kotlin/Native contains a single background thread.
+This is the dispatcher that used by default in [GlobalScope]. 
```

Comment:
> This is the dispatcher that used by default in [GlobalScope].
Missing "is":
This is the dispatcher that **is** used by default in [GlobalScope].

Reply:
Thanks, fixed.